features/telemetry/protocols/bmp/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/bmp/parser/proto/bgp/packets/OpenPacket.java

```diff
@@ -66,8 +92,10 @@ public OpenPacket(final Header header, final ByteBuf buffer, final PeerFlags fla
 
         final int parametersLength = uint8(buffer);
 
-        // Skip the parameters (see https://tools.ietf.org/html/rfc4271#section-4.2)
-        skip(buffer, parametersLength);
+        final List<Parameter> list = repeatRemaining(slice(buffer, parametersLength), Parameter::new);
+
+        parameters = list.stream().filter(p -> p.type != 2).collect(Collectors.toList());
```

Comment:
I don't think we should filter the parameters, as capabilities are parameters, too. An we should keep the link to the rfc for further reference to there parameter types, if there are any.

Reply:
Ok